go-backend/db/pg.go

```diff
@@ -43,21 +49,25 @@ func Init() (s Storer, err error) {
 	return &pgStoreConn, nil
 }
 
+// RunMigrations - runs all database migrations (see ../migrtions/*.up.sql)
 func RunMigrations() (err error) {
 	uri := config.ReadEnvString("DB_URI")
 
 	db, err := sql.Open(dbDriver, uri)
 	if err != nil {
+		// TODO: Log sql.Open failure here
```

Comment:
is there a need for these `TODOS`. Caller function may already be logging this. If not we should log the message there only

Reply:
I use TODO comments in code to remind me of what needs to be handled later to "button up" the code and polish it for production use. So unless there's a problem with me using those, then I intend to continue doing so. As for the error handling, 37266ebff51dd72086e371bb899cf91d1176cc6c